keychain_unix.cpp

```diff
@@ -76,37 +82,68 @@ static DesktopEnvironment detectDesktopEnvironment() {
     return DesktopEnv_Other;
 }
 
-static KeyringBackend detectKeyringBackend()
+static KeyringBackend detectKeyringBackend(void)
```

Comment:
another useless "void"

Reply:
ah, old `C` habits, always forget that the definition in `C++` differs ^^